### PR TITLE
added factorial program using for loop

### PR DESCRIPTION
**Pull request by Pratik180198**
Please add the label "Hactoberfest"

### CODE REVIEW

**Review by pythonites — CHANGES_REQUESTED** [review]
Hello @Pratik180198 
As this is a beginner level repo, I suggest you to please add comments to your program.
Though hacktoberfest-accepted level can be added but making the necessary changes will be good.

Thank you for the contribution.
Regards @pythonites

**Comment by Pratik180198** [comment]
Can you please help me how to changes request in the github

**Comment by pythonites** [comment]
> Can you please help me how to changes request in the github

I think [this](https://stackoverflow.com/questions/9790448/how-to-update-a-pull-request-from-forked-repo) is what you are looking for.

File: factorial_01.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+num1=input("Enter number: ")
+try:
```
Comment (Pratik180198) [inline factorial_01.py:2]: #The try block lets you test a block of code for errors.

File: factorial_01.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+    num=int(num1)
+    fact=1
+    for i in range(1,num+1): #Using For Loop
+        fact=fact*i
+    print("Factorial of {} is {}".format(num,fact)) #For Loop Answer
+except:
```
Comment (Pratik180198) [inline factorial_01.py:8]: #The except block lets you handle the error.

**Review by Pratik180198 — COMMENTED** [review]
Try and except comment added

**Comment by Pratik180198** [comment]
Sir please review that changes are right.

**Review by pythonites — APPROVED** [review]

**Pull request merged**